go/private/go_toolchain.bzl

```diff
@@ -57,6 +63,7 @@ go_toolchain_core_attrs = {
     "cgo_link_flags": attr.string_list(default=[]),
     "goos": attr.string(mandatory = True),
     "goarch": attr.string(mandatory = True),
+    "_external_linker": attr.label(default=_get_linker),
```

Comment:
Why not inline `_get_linker`?

Reply:
The point of this change was for _external_linker to become a computed dependency, where it reduces to None if there is no external linker available, thus breaking the dependency on the cpp fragment where you are in pure go mode.